src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -27,11 +28,16 @@ internal class FunctionExecutor : IFunctionExecutor
         private readonly IWebJobsExceptionHandler _exceptionHandler;
         private readonly TraceWriter _trace;
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _fastLogger;
+        private readonly ILogger _logger;
```

Comment:
And yes, this is one of the most confusing classes wrt logging. There's so much going on here that we'll need to pull some of it in the next major release.